docs/hub/spaces-circleci.md

```diff
@@ -0,0 +1,58 @@
+# Managing Spaces with CircleCI Workflows
```

Comment:
You'll also need to add this to https://github.com/huggingface/hub-docs/blob/main/docs/hub/_toctree.yml for it to show in the table of contents

Reply:
Thanks for showing that. Updated with another commit 👍🏽